api/routes.go

```diff
@@ -27,3 +34,23 @@ func (api *API) healthGET(w http.ResponseWriter, req *http.Request, _ httprouter
 		SkydAlive: ph.Skyd == nil,
 	})
 }
+
+// userAddressPOST is the handler for the /address endpoint.
+func (api *API) userAddressPOST(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	// Get sub from accounts service.
+	sub, err := api.staticPromoter.SubFromAuthorizationHeader(req.Header.Get("Authorization"))
```

Comment:
Hm... I suppose we get the cookie header here as well, right? In that case we should pass that one as well. Most people will be authenticated using cookies, rather than `auth` header. The cookie header contains encrypted data but `accounts` will accept it without issues.